src/backend/DocDownloaderRedisCache/pom.xml

```diff
@@ -19,6 +19,8 @@
         <jedis.version>5.1.2</jedis.version>
         <joda-time.version>2.12.7</joda-time.version>
         <splunk-library-javalogging.version>1.11.8</splunk-library-javalogging.version>
+        <logback-classic>1.4.14</logback-classic>
+        <janino>3.1.6</janino>
```

Comment:
Just curious Peiman, what is Janino used for?

Reply:
If it is missing the logback initialization will fail, my guess is that it is used internally with logback logic condition that separates the logging with/without splunk via env variable SPLUNK_URL.